lib/absinthe/schema/rule.ex

```diff
@@ -1,40 +0,0 @@
-defmodule Absinthe.Schema.Rule do
```

Comment:
@benwilson512 Was this leftover from before Schema phases?

Reply:
Yes, it can be safely deleted.